core/solver/idr.cpp

```diff
@@ -170,6 +170,25 @@ void Idr<ValueType>::iterate(const LinOp *b, LinOp *x) const
 
     int total_iter = -1;
 
+    /* Memory movement summary for iteration with subspace dimension s
+     * Per iteration:
+     * (11/2s+10+5/(s+1))n * values + matrix/preconditioner storage
+     * For (s+1) iterations:
```

Comment:
I counted the number of SpMVs, to more closely match what GMRES is doing. This is a bit inconsistent (should be s iterations, not s+1), but since it is difficult to rewrite IDR in a restart-like fashion since we don't have a recurrent residual estimate, I left it like this for now.